### ISSUE TITLE
Bug: 3rd & 4th player 

### ISSUE THREAD

**Issue by Tmac4930** [luke-goldfain/Hangtime-Prototype#50]

**Comment by luke-goldfain** [comment]
Please describe the bug! "3rd & 4th player camera functionality" is not enough for me to know what the issue is.

**Comment by Tmac4930** [comment]
Yeah I know, I'm gonna have to update this a bit later once I'm sure what the issue is. For the most part  it's the problem that we had with the third and fourth player controllers not connecting. However my notes where kinda rushed on the matter so once I'm a little less tired I'll try to put things into words.

**Comment by luke-goldfain** [comment]
Ping... Any update on this?

**Comment by Tmac4930** [comment]
when players are joining the game they may run into a problem that prevents them from joining if they are the third or fourth player to do so. This has been experienced multiple times, often resulting in us shutting off the game and turning it on again before trying again.

**Comment by luke-goldfain** [comment]
Yeah this is a weird issue that I couldn't identify the cause of. It expands to the second player sometimes too and happens seemingly for no reason at all. Definitely needs to be looked into.

**Comment by luke-goldfain** [comment]
No progress on the cause of or solutions for this issue, but it's worth noting that unplugging and replugging all controllers _while_ the game is running causes controllers to work as expected. This will have to be our working solution for the events we'll be attending.
